build-xcframework.sh

```diff
@@ -540,18 +540,18 @@ combine_static_libraries "build-tvos-device" "Release-appletvos" "tvos" "false"
 # Create XCFramework with correct debug symbols paths
 echo "Creating XCFramework..."
 
-    if [[ "${BUILD_STATIC_XCFRAMEWORK}" == "ON" ]]; then
-        xcodebuild -create-xcframework \
-            -framework $(pwd)/build-ios-sim/framework/whisper.framework \
-            -framework $(pwd)/build-ios-device/framework/whisper.framework \
-            -framework $(pwd)/build-macos/framework/whisper.framework \
-            -framework $(pwd)/build-visionos/framework/whisper.framework \
-            -framework $(pwd)/build-visionos-sim/framework/whisper.framework \
-            -framework $(pwd)/build-tvos-device/framework/whisper.framework \
-            -framework $(pwd)/build-tvos-sim/framework/whisper.framework \
-            -output $(pwd)/build-apple/whisper.xcframework
-        exit 0
-    fi 
+if [[ "${BUILD_STATIC_XCFRAMEWORK}" == "ON" ]]; then
+    xcodebuild -create-xcframework \
+        -framework $(pwd)/build-ios-sim/framework/whisper.framework \
+        -framework $(pwd)/build-ios-device/framework/whisper.framework \
+        -framework $(pwd)/build-macos/framework/whisper.framework \
+        -framework $(pwd)/build-visionos/framework/whisper.framework \
+        -framework $(pwd)/build-visionos-sim/framework/whisper.framework \
+        -framework $(pwd)/build-tvos-device/framework/whisper.framework \
+        -framework $(pwd)/build-tvos-sim/framework/whisper.framework \
+        -output $(pwd)/build-apple/whisper.xcframework
+    exit 0
+fi 
```

Comment:
Nit: can you remove the extra whitespace after `fi` (does not show up in github but is visible using a local git diff). You can add `[no ci]` to the commit to avoid running the full CI again.